pom.xml

```diff
@@ -195,7 +195,8 @@
     <!-- Please don't upgrade the version to 4.10+, it depends on JDK 11 -->
     <antlr4.version>4.9.3</antlr4.version>
     <jpam.version>1.1</jpam.version>
-    <selenium.version>3.141.59</selenium.version>
+    <selenium.version>4.2.2</selenium.version>
+    <htmlunit-driver.version>3.62.0</htmlunit-driver.version>
```

Comment:
Huh, so we're using two different versions of different htmlunit components? what if all of htmlunit is updated?

Reply:
No, `htmlunit-driver` is in `SeleniumHQ`, the latest version is `3.64.0`, `htmlUnit` is in by `HtmlUnit`, and the latest version is `2.64.0`

- https://github.com/SeleniumHQ/htmlunit-driver/releases
- https://github.com/HtmlUnit/htmlunit/releases

Reply:
Got it, and can it match the selenium version then or are they released pretty differently?

Reply:
This is a different project maintained by two organizations, the major versions are different

Reply:
Yes, they match the version of selenium